docs/iris/src/whatsnew/contributions_2.3.0/newfeature_2019-Oct-02_cube_equality_tolerance.txt

```diff
@@ -0,0 +1,3 @@
+* Cube data equality testing (and hence cube equality) now uses a more relaxed
+  tolerance, thus some cubes may now test 'equal' that previously did not.
+  The new behaviour is the default for :func:`numpy.allclose`.
```

Comment:
If we are choosing to go down this path of relaxing the tolerance, I think it would helpful to our users to explain more specifically what has changed.
As a user, I would like to know what the behaviour was before (i.e. only absolute tolerance of 1e-8) and what the behaviour is now (as you have done with linking of to allclose

Reply:
Ok!